app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "How to choose your teacher training course"
+subcategory: Postgraduate teacher training
+description: |-
+  Find out how to choose your postgraduate teacher training course based on your eligibility, the qualifications it awards, and the course provider.
+related_content:
+    Choosing the right teacher training course provider : "/blog/choosing-the-right-teacher-training-course-provider" 
+    Train to teach if you have a degree : "/train-to-be-a-teacher/if-you-have-a-degree"
+    Your teacher training application : "/how-to-apply-for-teacher-training/teacher-training-application"
+external_content:
+    - content/train-to-be-a-teacher/promos/find-your-course
+promo_content:
+    - content/train-to-be-a-teacher/promos/adviser-promo-degree
+navigation: 20.25
+navigation_title: How to choose your course
+navigation_description: Find out how to choose your postgraduate teacher training course.
+keywords:
+  - becoming a teacher
+  - teacher training
+  - pgce
+  - qts
+  - Teach First
+  - Postgraduate teaching apprenticeship
+
+---
+
+All postgraduate teacher training courses include time spent in school placements with some theoretical learning. 
+
+The main differences between courses are: 
+
+* what you need to get onto the course 
+* the qualifications the course awards 
+* if the course comes with a salary 
+* the course provider 
+* if it’s full or part-time 
+
+## What you need to get onto the course 
+
+Some training courses have specific eligibility requirements. 
+
+It's important to check the requirements for each course you apply to, so you do not waste time on an application you’re unlikely to be successful with. 
+
+### Your qualifications 
+
+All postgraduate teacher training courses require you to have: 
+
+* a degree in any subject 
+* GCSEs at grade 4 (C) or above in English and maths (and science if you want to teach primary) 
+
+Most course providers will accept a degree grade of 2:2 or higher, but some may require you to have a 2:1. 
+
+Check each course provider’s requirements on their course page. You can also [find out more about the qualifications you need to teach](/is-teaching-right-for-me/qualifications-you-need-to-teach). 
+
+### Your right to work in the UK 
+
+You’ll need the right to work or study in the UK to do your teacher training in England. 
+
+Some training providers offer courses with visa sponsor available. You can filter by ‘visa sponsorship’ to [find courses where visas can be sponsored](https://www.find-postgraduate-teacher-training.service.gov.uk/).   
```

Comment:
Does this sentence need some context beforehand? Eg 'If you are a non-UK citizen (and you don't have the right to work???)?....

I'm not sure what the right wording is but I think as it stands, it's not clear who this is aimed at.